twoliter/embedded/Makefile.toml

```diff
@@ -540,6 +541,31 @@ fi
 '''
 ]
 
+# Task to lint Bottlerocket Security Advisories by checking for non-ASCII characters
+# and verifying that each versioned directory under "advisories" has a corresponding
+# tag on the Twoliter project this task runs against.
+[tasks.check-advisories]
+script_runner = "bash"
+script = [
+'''
+if find advisories -name '*.toml' -type f >/dev/null 2>&1 ; then
+  export LC_ALL=C
+  if grep --include '*.toml' -R -l -P ['\x80'-'\xFF'] advisories ; then
+    echo "error: found non-ASCII characters in advisories" >&2
+    exit 1
+  fi
+
+  for version in $(find advisories/* -type d -not -path advisories/staging); do
+    grep v$(basename ${version})$ <(PAGER= git tag)
+    if [ "$?" -ne 0  ]; then
```

Comment:
This implies a dependency on the project this check runs against having all tags fetched. Curious for feedback on either:
* fetching tags in this check
* or, removing this as a default check and running this check as a new GitHub CI step on projects. Those projects will set `fetch-tags: true` in respective `actions/checkout` steps